recipes/libhugetlbfs/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "libhugetlbfs" %}
+{% set version = "2.23" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/libhugetlbfs/libhugetlbfs/releases/download/{{ version }}/libhugetlbfs-{{ version }}.tar.gz
+  sha256: b70672f1e807c61b7eb3adf41c1903b42917951f2e7f8aef6821841700c04479
+
+build:
```

Comment:
Should there be a run_exports here?

Reply:
I added one thanks!

Reply:
Is there a way to automatically launch migrations?

Reply:
The bot will make automatic PRs on a new version to conda-forge-pinning if it is listed in the global pinning.